src/main/java/seedu/address/logic/parser/AddCategoryCommandParser.java

```diff
@@ -52,12 +56,15 @@ public AddCategoryCommand parse(String args) throws ParseException {
 
     }
 
-    /**
-     * Returns true if none of the prefixes contains empty {@code Optional} values in the given
-     * {@code ArgumentMultimap}.
-     */
-    private static boolean arePrefixesPresent(ArgumentMultimap argumentMultimap, Prefix... prefixes) {
-        return Stream.of(prefixes).allMatch(prefix -> argumentMultimap.getValue(prefix).isPresent());
+    private boolean hasDuplicates(List<String> list) {
```

Comment:
Nice fix for duplicates. Will be using the same method for edit. Rmbr to add in assertion.